pkg/operation/botanist/etcd.go

```diff
@@ -65,23 +70,16 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 			DefragmentationSchedule:     &defragmentationSchedule,
 			CARotationPhase:             v1beta1helper.GetShootCARotationPhase(b.Shoot.GetInfo().Status.Credentials),
 			RuntimeKubernetesVersion:    b.Seed.KubernetesVersion,
+			HVPAEnabled:                 hvpaEnabled,
+			MaintenanceTimeWindow:       *b.Shoot.GetInfo().Spec.Maintenance.TimeWindow,
+			ScaleDownUpdateMode:         getScaleDownUpdateMode(class, b.Shoot),
 			PriorityClassName:           v1beta1constants.PriorityClassNameShootControlPlane500,
 			HighAvailabilityEnabled:     v1beta1helper.IsHAControlPlaneConfigured(b.Shoot.GetInfo()),
 			TopologyAwareRoutingEnabled: b.Shoot.TopologyAwareRoutingEnabled,
+			VPAEnabled:                  features.DefaultFeatureGate.Enabled(features.VPAForETCD),
```

Comment:
Maybe add unit test to verify when the feature gate is enabled, this field will be true and vice versa?

Reply:
I'm not sure, what would this be testing? I don't see tests for any of the other featuregates either.

Reply:
Ok, I was also unsure writing this comment. So, let's keep it as it is.